pocketimdb/movies/serializers.py

```diff
@@ -1,5 +1,24 @@
 from rest_framework import serializers
-from .models import Movie, MovieLike
+from .models import Movie, MovieLike, MovieComment
+from .utils import get_timestamp_relative_diff
+
+class AddMovieCommentSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = MovieComment
+        fields = ['comment']
+
+class MovieCommentSerializer(serializers.ModelSerializer):
+
+    content = serializers.CharField(source='comment')
+    user = serializers.StringRelatedField(read_only=True)
```

Comment:
ovdje nam je bolje da korisimo `PrimaryKeyRelatedField` iz razloga sto ovako ako ostavis i overrideujes metodu `__str__` on ce promjeniti i nacin vracanja podatka, a ako stavis PrimaryReleatedFIeld on ce uvijek vracati id tog odela

Reply:
U sustini to ima mnogo vise smisla nego da se koristi StringRelatedField, sada to vidim. Ali opet, on ce meni vracati id user-a a ja zelim da prikazem username korisnika tj. u mom slucaju email, a ne njegov id. Tako da mi treba neka kombinacija ta dva resenja,  tj da konkretno email polje dovlacim.

Reply:
onda napravis novi serialzer koji ce da bude tipa CommentUserSerialzer i posmatras iz perspektive user modela, pa onda kada koristis ovdje onda korisits ovako 
```
content = ...
 user = CommentUserSerialzer()
```

Reply:
Uradio sam tako nesto bas malo pre, napravio serializer u user modulu koji vraca samo email